cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2004,3 +2004,74 @@ action.summary_index = 1
+                  Cyences add-on for Splunk -> sudousers.sh
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_linux_users_with_previledge`\
+| table _time host UID USERNAME GID HOME_DIR \
+| eval user_discovery=_time \
+| stats earliest(_time) as _time latest(*) as * by host UID \
+| append \
+    [| search  `cs_linux_sudousers`\
+    | stats latest(_raw) as raw latest(_time) as sudo_discover by host \
+    | rex field=raw "sudouser=(?<USERNAME>.*)" max_match=0 \
+    | fields - raw \
+    | mvexpand USERNAME \
+    | eval sudo_access="Yes" ] \
+| stats first(sudo*) as sudo*_new first(user_discovery) as user_discovery_new first(*) as * by host, USERNAME \
+| appendpipe \
+    [| inputlookup user_list.csv append=T ] \
+| stats first(*) as * by host USERNAME \
+| eval sudo_last_modified=case(isnull(sudo_access_new) and isnull(sudo_access),null(),isnotnull(sudo_access_new) and isnull(sudo_access),sudo_discover_new, isnull(sudo_access_new) and isnotnull(sudo_access),now(),isnotnull(sudo_access_new) and sudo_access=="User's Sudo Access Revoked OR Splunk is not Receiving Logs from Host",sudo_discover_new,sudo_access_new==sudo_access,sudo_last_modified) \
+| eval sudo_access=case(isnull(sudo_access_new) and isnull(sudo_access),null(),isnotnull(sudo_access_new) and isnull(sudo_access),sudo_access_new, isnull(sudo_access_new) and isnotnull(sudo_access),"User's Sudo Access Revoked OR Splunk is not Receiving Logs from Host",isnotnull(sudo_access_new) and sudo_access=="User's Sudo Access Revoked OR Splunk is not Receiving Logs from Host",sudo_access_new,sudo_access_new==sudo_access,sudo_access) \
+| eval user_status=if(isnull(user_discovery_new),"Local User has been Removed OR Splunk is not receiving logs from Host","Available") \
+| eval user_last_modified=case(isnull(user_discovery_new) ,now(),isnotnull(user_last_modified), user_last_modified,true(),user_discovery_new)\
+| fields - sudo_access_new,sudo_discover_new,user_discovery_new \
+| outputlookup user_list.csv
+
+
+[Linux/Unix - Change in Sudo Access of Linux Local Account]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 59 * * * *
+description = When change in Sudo access of Linux/Unix Local User Account\
+\
+Data Collection - Report: "Linux : Local User List Lookup Gen" is enabled.
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = | inputlookup user_list.csv | where sudo_last_modified>relative_time(now(),"-24h")
```

Comment:
This would generate multiple alerts (keep it -60m)
- one change would generate 24 alerts.